### DISCUSSION TITLE
User Experience Enhancements

### DISCUSSION THREAD

**Discussion by adesokan01** [adesokan01/T24S2T1EXTG01#7] [General]
What additional features can we implement to enhance user experience in the currency converter? Should we include options for saving frequently used currencies, or adding a 'swap currencies' button to quickly interchange the 'From' and 'To' fields?

**Comment by acantley** [comment]
Functionality to interchange between 'From' and 'To' would definitely enhance UX. Saving frequently used currencies (or favorites) would require complex logic that given our project's time constraints may not be feasible for now.

I like the ideas though, and they could inform a subsequent version update of the currency calculator.
